build/ci/updater/pom.xml

```diff
@@ -90,18 +90,52 @@
       <artifactId>mssql-jdbc</artifactId>
       <version>9.4.1.jre11</version>
     </dependency>
-    <dependency>
+    <dependency> <!-- OF-2960: Including dependencies of ojdbc11-production rather than ojdbc11-production itself. -->
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc11</artifactId>
+      <version>${ojdbc.version}</version>
+    </dependency>
```

Comment:
This fails in the tests because my hacky CI Updater test rig isn't a child project of the parent POM. Should we make it be a child? No clue if there'd be side effects.